lib/music/music_album.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../music_item'
+
+class MusicAlbum < MusicItem
+  attr_accessor :on_spotify
+
+  def initialize(publish_year, on_spotify: false)
+    super(publish_year, is_archived: false)
+    @on_spotify = on_spotify
+  end
+
+  def can_be_archived?
+    super && @on_spotify
+  end
```

Comment:
optional
Please check in UML diagram this method is private(not public)

Reply:
And it is private in my case too, check ``music_item.rb``